README.md

```diff
@@ -28,13 +28,13 @@ Quick Start
 
 #### Installation
 
-1. Install the vimarsh app
+1. Install the yaksh_app
     
-        pip install vimarsh
+        pip install yaksh_app
```

Comment:
The name of the package is Yaksh
The django project can be yaksh-app or whatever.